torchao/float8/config.py

```diff
@@ -98,7 +102,7 @@ class Float8GemmConfig:
     use_fast_accum: bool = False
 
 
-@dataclass(frozen=True)
+@dataclass(frozen=False)
```

Comment:
Why did we unfreeze?

Reply:
Ahh because of the post init update

Reply:
we might need to change configs depending on what happens with torchao level training API, I'm hoping to bundle all bc-breaking changes into that to minimize # of bc-breaking changes